src/store/actions/user.actions.ts

```diff
@@ -70,3 +71,10 @@ export const setActiveUser = (userId: string): ThunkAction<Promise<void>, RootSt
       .catch(() => dispatch({type: SET_ACTIVE_USER_FAILED}));
   };
 };
+
+export const removeActiveUser = (): ThunkAction<Promise<void>, RootState, unknown, Action> => {
```

Comment:
rename this to logout, and rename the setActiveUser to login. also do this for all the types associated with these actions.

Reply:
@nklomp @Brummos I guess ... making it logout is not right ... Let's say I logged out ... and now want to login with a different user. We have not designed the wallet to handle that... I suggest to keep it the same for now.

In the scope of this 3 story point ticket doing that kind of refactor will increase the testing surface too big ... even the VC and contacts may need to be checked ...

Reply:
It is just a name suggestion, as the removeUser is highly confusing

Reply:
renamed